redaxo/src/core/lib/var/var.php

```diff
@@ -409,7 +409,7 @@ private function getGlobalArgsOutput()
      */
     public static function toArray($value)
```

Comment:
laut phpdoc ist gar kein `null` erlaubt für den parameter. weißt du wo der null-wert herkommt?

Reply:
War wohl eine Bug in meinem Code. Danke für den Hinweis zum PHP Doc.